prac_07/languages.csv

```diff
@@ -0,0 +1,6 @@
+Language,Typing,Reflection,Year
```

Comment:
Your other column for the pointer arithmetic and the datasets?
and I also believe that the requirement for you is to add another language piece of information to the file.